module/Decision/view/decision/member/index.phtml

```diff
@@ -0,0 +1,133 @@
+        <tr>
+            <th><?= $this->translate('Membership number') ?></th>
+            <td><?= $member->getLidnr() ?></td>
+        </tr>
+        <tr>
+            <th><?= $this->translate('Initials') ?></th>
+            <td><?= $member->getInitials() ?></td>
+        </tr>
+        <tr>
+            <th><?= $this->translate('First name') ?></th>
+            <td><?= $member->getFirstName() ?></td>
+        </tr>
+        <tr>
+            <th><?= $this->translate('Middle name') ?></th>
+            <td><?= $member->getMiddleName() ?></td>
+        </tr>
+        <tr>
+            <th><?= $this->translate('Last name') ?></th>
+            <td><?= $member->getLastName() ?></td>
+        </tr>
+        <tr>
+            <th><?= $this->translate('Sex') ?></th>
+            <td><?php
+switch ($member->getGender()) {
+case Member::GENDER_MALE:
+    echo $this->translate('Male');
+    break;
+case Member::GENDER_FEMALE:
+    echo $this->translate('Female');
+    break;
+case Member::GENDER_OTHER:
+    echo $this->translate('Other');
+    break;
+}
+?></td>
+        </tr>
+        <tr>
+            <th><?= $this->translate('Email') ?></th>
+            <td><a href="mailto:<?= $member->getEmail() ?>"><?= $member->getEmail() ?></a></td>
+        </tr>
+        <tr>
+            <th><?= $this->translate('Birth date') ?></th>
+            <td><?= $member->getBirth()->format('l j F Y') ?></td>
+        </tr>
+        <tr>
+            <th><?= $this->translate('Generation') ?></th>
+            <td><?= $member->getGeneration() ?></td>
+        </tr>
+        <tr>
+        <th><?= $this->translate('Membership type') ?></th>
+            <td><?php
+switch ($member->getType()) {
+case Member::TYPE_ORDINARY:
+    echo $this->translate('Ordinary');
+    break;
+case Member::TYPE_PROLONGED:
+    echo $this->translate('Prolonged');
+    break;
+case Member::TYPE_EXTERNAL:
+    echo $this->translate('External');
+    break;
+case Member::TYPE_EXTRAORDINARY:
+    echo $this->translate('Extraordinary');
+    break;
+case Member::TYPE_HONORARY:
+    echo $this->translate('Honorary');
+    break;
+}
+?></td>
+        </tr>
+        <tr>
+            <th><?= $this->translate('Last membership change') ?></th>
+            <td><?= $member->getChangedOn()->format('l j F Y') ?></td>
+        </tr>
+        <tr>
+            <th><?= $this->translate('End date of membership') ?></th>
+            <td><?= $member->getExpiration()->format('l j F Y') ?></td>
+        </tr>
+    </table>
+    <h3><?= $this->translate('Membership of committees and fraternities') ?></h3>
+<ul>
+<?php foreach ($member->getOrganInstallations() as $install): ?>
+    <li><?= $install->getFoundation()->getAbbr() ?></li>
+<?php endforeach; ?>
+</ul>
+</div>
+<div class="col-md-6">
+<h3><?= $this->translate('Adresses') ?></h3>
+<?php $addresses = array() ?>
+<?php foreach ($member->getAddresses() as $address): ?>
+<?php $addresses[$address->getType()] = $address ?>
```

Comment:
Oops. Helemaal waar inderdaad. Is een overblijfsel van de vergelijkbare pagina in de database.